src/modules/add-licences/controller.js

```diff
@@ -321,6 +318,17 @@ const getLicence = async documentId => {
   return data;
 };
 
+const updateFormForInvalidAddress = (form) => {
```

Comment:
We can remove the need for bespoke validation by creating a dynamic schema for both forms